server/@types/hmpo-form-wizard/index.d.ts

```diff
@@ -70,9 +71,10 @@ declare module 'hmpo-form-wizard' {
     NotSpecified = 9,
   }
 
-  function FormWizard(steps: Steps, fields: Fields, config: FormWizardConfig)
+  function FormWizard(steps: FormWizard.Steps, fields: FormWizard.Fields, config: FormWizardConfig)
 
   namespace FormWizard {
```

Comment:
Yeah I copied the changes to the form wizard and types etc from the Locations repo. I spoke to them and they said they have done a lot of work and made changes since working with it more. I also thought these updated and more vigorous types will come in useful when we try to automate their creation as I am look at now.

Reply:
oh absolutely! i love stricter, more accurate types! would be good to update the comment at the top to say where it came from